subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/alignment/AlignmentIntegrationTest.groovy

```diff
@@ -1119,6 +1119,7 @@ class AlignmentIntegrationTest extends AbstractAlignmentSpec {
     }
 
     @Issue("gradle/gradle#7916")
+    @ToBeFixedForInstantExecution(because = ":dependencyInsight")
```

Comment:
We should probably add issues to the board to fix these tasks. I wonder if they belong in one of the milestones?